### PR TITLE
Share location with alert

### PR DESCRIPTION
**Pull request by RutsSantos**
#### Description:
When you're confirmed covid-19 positive, a dialog asking you to share you location with the health authorities pops up, if you accept your location starts to share. It can be toggled on or of from Settings.

#### Screenshots:
Simulator Screen Shot - iPhone 11 - 2020-06-18 at 15 42 06
Simulator Screen Shot - iPhone 11 - 2020-06-23 at 14 27 55

#### How to test:
Insert a positive ID in 'Report' or in 'Covid positive support' and test.

### CODE REVIEW

File: app/services/LocationService.js [hunk 1]
```diff
@@ -224,7 +220,7 @@ export default class LocationServices {
               return data;
             })
             .catch(error => {
-              console.error('[ERROR] ' + error);
+              console.error('[ERRORrrr] ' + error);
```
Comment (D-Rosa99) [inline app/services/LocationService.js]: why did you put "rrr" ??
```suggestion
              console.error('[ERROR] ' + error);
```

File: app/locales/es.json [hunk 2]
```diff
@@ -137,6 +137,8 @@
     "location_match_description": "Comprueba si has tenido contacto con alguien infectado",
     "logging_active": "Localización activada",
     "logging_inactive": "Localización desactivada",
+    "share_loc_active": "Estás compartiendo tu ubicación con el Ministerio de Salud",
```
Comment (D-Rosa99) [inline app/locales/es.json]: The resources are order alphabetically, please follow the convention and order your resource alphabetically

File: app/locales/es.json [hunk 3]
```diff
@@ -137,6 +137,8 @@
...
+    "share_loc_inactive": "No estás compartiendo tu ubicación con el Ministerio de Salud",
```
Comment (D-Rosa99) [inline app/locales/es.json]: follow the convention and order your resource alphabetically, same in english

File: app/services/LocationService.js (hunk 1 above)
Comment (RutsSantos) [reply app/services/LocationService.js]: It was just a test thing, forgot to delete it

**Review by inteljudajocu — APPROVED** [review]
LGTM

**Review by JoseFcoEstrella — APPROVED** [review]

**Review by D-Rosa99 — APPROVED** [review]

**Pull request merged**